.gitignore

```diff
@@ -16,6 +16,8 @@ bin/
 gen/
 out/
 
+kotlin-js-store
```

Comment:
I think this should be in the repo and not ignored based on comments in Slack from JB engineers.

Reply:
I am also in favor. Adding the kotlin-js-store hasn't been added consistently to all arrow projects. 
Maybe it makes sense to decide, whether we should have it in all projects or not
cc @raulraja , @nomisRev

Reply:
> based on comments in Slack from JB engineers.

If it's officially recommended to keep this file in git then I think we should.

Reply:
Will add it :D and do the same for the proofs and analysis plugin